functions/helloworld/hello-gcs/src/main/java/functions/eventpojos/GcsEvent.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions.eventpojos;
+
+import java.util.Date;
+
+// [START functions_helloworld_gcs_event]
+public class GcsEvent {
+  // Cloud Functions uses GSON to populate this object.
```

Comment:
Note - I'm Ok w/ this, but in the future, you might wish to consider using AutoValue, or adding hashCode, tostring, and equals to truely be a POJO.

Reply:
Fair point - these are (ideally) temporary classes anyway; see b/153731328 for tracking + more info.